### ISSUE TITLE
TypeError :nonetype object is not callable .

### ISSUE THREAD

**Issue by Rainbow-LollipopP** [iCVTEAM/PGNet#4]
![image](https://user-images.githubusercontent.com/105594024/170057115-d57c4ff7-5093-4af7-bcb0-45222dc7e2d8.png)

**Comment by Rainbow-LollipopP** [comment]
是数据集加载不对吗，我将下载的UHRSD-2K数据集重名为range里面的第一个DAVIS-S

**Comment by xiechenxi99** [comment]
> 是数据集加载不对吗，我将下载的UHRSD-2K数据集重名为range里面的第一个DAVIS-S
self.loader = DataLoader(self.data, batch_size=1, shuffle=False, num_workers=8) 可以把8改成0试一下

**Comment by Rainbow-LollipopP** [comment]
> > 是数据集加载不对吗，我将下载的UHRSD-2K数据集重名为range里面的第一个DAVIS-S
> > self.loader = DataLoader(self.data, batch_size=1, shuffle=False, num_workers=8) 可以把8改成0试一下

 改了的，但是还是出现这个问题，debug了发现是dataloader读不到数据，也就是没数据所有显示为Nonetype 貌似是读取数据路径部分出了问题

**Comment by Rainbow-LollipopP** [comment]
> > 是数据集加载不对吗，我将下载的UHRSD-2K数据集重名为range里面的第一个DAVIS-S
> > self.loader = DataLoader(self.data, batch_size=1, shuffle=False, num_workers=8) 可以把8改成0试一下
![image](https://user-images.githubusercontent.com/105594024/170303582-cca84f48-0c70-43a4-8abe-e8af29a6b190.png)
第一个问题找不到路径解决是在加载模型前面多加一个.就可以了
![image](https://user-images.githubusercontent.com/105594024/170303544-8eace814-c634-4bba-931e-825c35d59799.png)
nonetype问题则是在dataset.py里面加载数据时，name=\\mask0000x.png,在下面imread读取时找不到此路径，加入一句name=name.spilt('\\')[1]即可以解决

**Comment by xiechenxi99** [comment]
> > > 是数据集加载不对吗，我将下载的UHRSD-2K数据集重名为range里面的第一个DAVIS-S
> > > self.loader = DataLoader(self.data, batch_size=1, shuffle=False, num_workers=8) 可以把8改成0试一下
> > > ![image](https://user-images.githubusercontent.com/105594024/170303582-cca84f48-0c70-43a4-8abe-e8af29a6b190.png)
> > > 第一个问题找不到路径解决是在加载模型前面多加一个.就可以了
> > > ![image](https://user-images.githubusercontent.com/105594024/170303544-8eace814-c634-4bba-931e-825c35d59799.png)
> > > nonetype问题则是在dataset.py里面加载数据时，name=\mask0000x.png,在下面imread读取时找不到此路径，加入一句name=name.spilt('\')[1]即可以解决

👍
